pkg/controller/nodemaintenance/generated_mock_nodemaintenance_controller.go

```diff
@@ -1,49 +0,0 @@
-// Code generated by MockGen. DO NOT EDIT.
```

Comment:
Did you forget to re-add this for your tests?

Reply:
the test right now  isn't using the generated mock;

Reply:
the tests don't need the mock right now;  Yanir uses the mock to inject his informer trick.